src/JustEat.StatsD/PublicAPI.Shipped.txt

```diff
@@ -1,4 +1,19 @@
 #nullable enable
+JustEat.StatsD.Buffered.Tags.InfluxDbTagsFormatter
```

Comment:
We should probably move these out of the `Buffered` namespace and into either the root namespace, or a new one called something like `TagFormatters`?

Reply:
I have moved them to the folder `TagsFormatters` matching the namespace `JustEat.StatsD.TagsFormatters`.